corehq/apps/users/device_rate_limiter.py

```diff
@@ -0,0 +1,93 @@
+from datetime import datetime, timezone
+
+from django.conf import settings
+from django_redis import get_redis_connection
+
+from corehq import toggles
+from corehq.apps.cloudcare.const import DEVICE_ID as CLOUDCARE_DEVICE_ID
+from corehq.util.metrics import metrics_counter, metrics_histogram
+
+DEVICE_RATE_LIMIT_MESSAGE = "Current usage for this user is too high. Please try again in a minute."
+DEVICE_SET_CACHE_TIMEOUT = 2 * 60  # 2 minutes
```

Comment:
This constant was the main thing that drew my attention when I was looking for the rate limit timeout, and it didn't occur to me that the key generation function was the more important place to look for that. What do you think of adding a comment here noting that this is purposefully longer than the 1-minute fixed window enforced by `DeviceRateLimiter._get_redis_key`?

Reply:
4c8c734ca860e30dca73c03cee2e9625f9748694